ProjectWithUMLDiagram/src/com/beyond/person/core/helper/Help.java

```diff
@@ -0,0 +1,25 @@
+package com.beyond.person.core.helper;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Help {
+    public static boolean checkAlphabet(String name) {
+        char[] ch = name.toCharArray();
+        for (char c : ch) {
+            if (!Character.isLetter(c)) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    public static boolean isEmailCorrect(String email){
```

Comment:
@AraChilingaryan name can be isEmailValid

Reply:
done